Sources/Spezi/Spezi.docc/Spezi Ecosystem.md

```diff
@@ -0,0 +1,268 @@
+# The Spezi Ecosystem
+
+<!--
+This source file is part of the Stanford Spezi open-source project
+SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+SPDX-License-Identifier: MIT
+-->
+
+Discover the core modules of the Spezi ecosystem and understand how you can use them to build your healthcare and research applications.
+
+
+## Module Overview
+This document outlines the modules specifically tailored to various use cases. 
+
+- [Spezi Onboarding](#spezi-onboarding) for user onboarding with welcome screens and more. 
+- [User Management](#user-management) for login account creation and customization flows.
+- [Spezi Contact](#spezi-contact) for managing and displaying contact information.
+- [Spezi AccessGuard](#spezi-accessguard) for login access codes and biometrics.
+- [Connected Devices and Spezi Bluetooth](#connected-devices-and-spezi-bluetooth) for supporting Bluetooth-connected devices.
+- [Cloud Systems](#cloud-systems) for Google Firebase integration and data synchronization.
+- [Local Data Storage](#local-data-storage) for efficient data persistence and retrieval.
+- [Electronic Health Integration](#electronic-health-integration) for managing FHIR data and data interoperability.
+- [Mobile Health Data](#mobile-health-data) for accessing HealthKit data.
+- [LLMs and AI Integration](#llms-and-ai-integration) for AI-driven functionalities.
+- [Spezi Questionnaire](#spezi-questionnaire) for customizable forms.
+- [Spezi Medication](#spezi-medication) for managing medication details, dosages, and more.
+- [Tasks and Reminders](#tasks-and-reminders) for task scheduling and notification management.
+- [Spezi Notifications](#spezi-notifications) for implementing user notifications and related.
```

Comment:
Some of the items in this list are names of specific modules (e.g. Spezi Onboarding), and some of them are names of features (e.g. Tasks and Reminders), which is a little confusing. I would either stick to one or the other. What do you think @PSchmiedmayer?

Reply:
Agree; I would suggest to stick with a generic setup and focus on the currently built out modules that have graduated beyond a 1.0. Anything that currently is not tagged with a 1.0 is not really considered stable and should probably not be listed here.